tests/api_e2e/anchor.test.js

```diff
@@ -9,7 +9,6 @@ const anchorPlaysPayload = require('../../fixtures/anchorPlays.json')
 const anchorPlaysByAgeRangePayload = require('../../fixtures/anchorPlaysByAgeRange.json')
 const anchorPlaysByAppPayload = require('../../fixtures/anchorPlaysByApp.json')
 const anchorPlaysByDevicePayload = require('../../fixtures/anchorPlaysByDevice.json')
-const anchorPlaysByEpisodePayload = require('../../fixtures/anchorPlaysByEpisode.json')
```

Comment:
Wondering why this test was removed. Is it related to the rest of the changes? 🤔

Reply:
#117